core/metadata.js

```diff
@@ -456,3 +459,34 @@ function buildFile (filePath /*: string */, stats /*: fs.Stats */, md5sum /*: st
   if ((stats.mode & EXECUTABLE_MASK) !== 0) { doc.executable = true }
   return doc
 }
+
+const CONFLICT_REGEXP = new RegExp('-conflict-\\d{4}(?:-\\d{2}){2}T(?:\\d{2}_?){3}.\\d{3}Z')
+module.exports.CONFLICT_REGEXP = CONFLICT_REGEXP
```

Comment:
Flow is unhappy about the way you mix `module.exports.whatever = ...` with existing `module.exports = { ... }` assignment in `core/metadata` :wink:

Reply:
I changed the export at the last minute: export a function rather than a RegExp object.